src/lib/index.ts

```diff
@@ -22,12 +22,23 @@ const main = async () => {
     const ghSha = process.argv.slice(2)[4];
     const ghPRNumber = process.argv.slice(2)[5] || '';
     let keepHistory = false;
+    let setPassIfNoBaselineFlag = false;
     let detailsLink = '';
-    if (process.argv.slice(2)[6] == 'keepHistory' || process.argv.slice(2)[7] == 'keepHistory') { 
-      keepHistory = true 
-    } else { 
-      detailsLink = process.argv.slice(2)[6] || ''
-    }
+
+    const options = process.argv.slice(8)
+
+    options.forEach(option => {
+      if (option === 'keepHistory')
+      {
+        keepHistory = true
+      } else if (option === 'setPassIfNoBaselineFlag')
```

Comment:
why else if? can't have keephistory and setPassIfNoBaselineFlag at the same time?

Reply:
oh true!

Reply:
no that is right, we loop through the options